### PR TITLE
Consistent naming: women and non-binary people, fixes #43

### LINKED ISSUE #43: make audience consistent: women and nonbinary people
as we've discussed before, women and nonbinary people is our intended audience. we should make this consistent in all the places.

### PR DESCRIPTION
**Pull request by vsmart**
Adresses https://github.com/clojurebridge-berlin/clojurebridge-berlin.github.io/issues/43

### CODE REVIEW

**Comment by vsmart** [comment]
@Malwine @socksy @lislis @Realtin What do you think?

File: index.html [hunk 1]
```diff
@@ -264,8 +264,8 @@
         </a>
       </div>
       <div class="col-md-6">
-        <p>ClojureBridge Berlin is organized with a lot of love by volunteers. We try our best to make the workshop a memorable event and bring more women* into programming.</p>
-        <p><i>ClojureBridge Berlin wird mit ganz viel Liebe von Freiwilligen organisiert. Wir versuchen unser Bestes, aus dem Workshop ein unvergessliches Event zu machen und mehr Menschen(FLTI) zum Programmieren anzuregen.</i></p>
+        <p>ClojureBridge Berlin is organized with a lot of love by volunteers. We try our best to make the workshop a memorable event and bring more people into programming.</p>
```
Comment (lislis) [inline index.html:267]: Maybe something that expresses the diversifying character of the workshop?
`and bring a divers group of people into programming`

**Comment by lislis** [comment]
@vsmart left a comment, but it's fine nonetheless :)

**Comment by Realtin** [comment]
does it make a difference saying 'women and nb people' vs. 'people identifing as nb and ...' ?

File: index.html (hunk 1 above)
Comment (vsmart) [reply index.html:267]: I considered that, but I think we stress that enough in other places. This paragraph is primarily about the fact that it's organised by volunteers & with a lot of love. Do you agree / does that make sense? 
(compare the German version: "menschen zum programmieren anregen")

Comment (lislis) [reply index.html:267]: That does make sense 💃

**Comment by lislis** [comment]
@Realtin I don't think so?

**Comment by vsmart** [comment]
I've seen "non-binary people" used in resources, e.g. here: http://gender.wikia.com/wiki/Non-binary
I think we're working on the common understanding that anyone who identifies as a woman, is a woman, and anyone who identifies as non-binary, is a non-binary person. Phrasing it that way is a bit shorter. 
Thsnks @lislis and @Realtin for the review!
Thanks @lislis for merging! :heart: :dancing_women: :sparkles:

**Comment by Malwine** [comment]
Thank you @vsmart 
bmo123

**Pull request merged**